src/cap32.cpp

```diff
@@ -152,6 +155,47 @@ double colours_green[32] = {
    0.2510, 0.3137, 0.5333, 0.5961
 };
 
+// added by a proposal from libretro project,
+// see https://github.com/ColinPitrat/caprice32/issues/135
+//
+
+double colours_green_libretro[32] = {
+   0.5755,  0.5755,  0.7534,  0.9718, 
+   0.1792,  0.3976,  0.4663,  0.6847, 
+   0.3976,  0.9718,  0.9136,  1.0300, 
+   0.3394,  0.4558,  0.6265,  0.7429, 
+   0.1792,  0.7534,  0.6952,  0.8116, 
+   0.1210,  0.2374,  0.4081,  0.5245, 
+   0.2884,  0.8626,  0.8044,  0.9208, 
+   0.2302,  0.3466,  0.5173,  0.6337
+};
+
+// the following formula would produce the table above
+/*
```

Comment:
One idea to "document" that is to have a unit test that contains this code and generates a table like the one above.
Then we compare the variable colours_green_libretro with the result of the code. What do you think?